src/Menu/Menu.scss

```diff
@@ -1,8 +1,51 @@
+$calorieTagColor: #49a0b5;
+$typeTagColor: #2e7cf6;
+$sideTagColor: #f7c244;
+
 .menu {
     &.show-restaurants {
         display: none;
     }
     &.show-menu {
         display: block;
     }   
-}
\ No newline at end of file
+}
+
+.food-item-card {
+    border: solid 2px #898c8e;
+    text-align: left;
+    box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.1), 0 6px 10px 0 rgba(0, 0, 0, 0.1);
+
+    h2 {
+        font-size: 22px;
+        font-weight: bold;
+    }
+    h3 {
+        font-size: 16px;
+        font-weight: bold;
+        border-radius: 5px;
+        padding: 1px 2px;
+        margin: 5px 0;
+    }
+    p {
+        font-size: 18px;
+        font-weight: bold;
+    }
+    .calories-tag {
+        background-color: $calorieTagColor;
+        color: #ffffff;
+    }
+    .tag-container {
+        display: flex;
```

Comment:
this styling could have been achieved with bootstrap, but is functional and it looks excellent!